sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -3116,16 +3116,17 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
           .getTableMetadata(TableIdentifier("tab1")).storage.locationUri.get)
 
         val fs = tablePath.getFileSystem(hadoopConf)
-        val trashRoot = fs.getTrashRoot(tablePath)
-        assert(!fs.exists(trashRoot))
+        val trashCurrent = new Path(fs.getTrashRoot(tablePath), "Current")
```

Comment:
Can we make a fake file system and use a temporary directory?

Reply:
+1 for @HyukjinKwon 's suggestion.

Reply:
Hii @HyukjinKwon, but `Trash.moveToAppropriateTrash` will always move the data to `trashCurrent`. can you guide me on what you mean by using a temporary directory instead.